tests/samplescli/project.pbxproj

```diff
@@ -623,4 +623,4 @@
 /* End XCConfigurationList section */
```

Comment:
I added a `large_project.pbxproj` as a sibling to this file in my branch. Not sure if A) that's where you'd want it, or B) if you even want it checked in (it's a 9MB file).

Reply:
Hm, I guess because the PR was already merged the new changes/files don't show up here, but the large project can be found at https://github.com/FuegoFro/mod-pbxproj/blob/faster_save/tests/samplescli/large_project.pbxproj